mkdocs.yml

```diff
@@ -5,44 +5,44 @@ nav:
       - Introduction: introduction.md
       - Common principles: common-principles.md
       - Modality agnostic files: modality-agnostic-files.md
-      - Modality specific files:
-        - Magnetic Resonance Imaging: modality-specific-files/magnetic-resonance-imaging-data.md
-        - Magnetoencephalography: modality-specific-files/magnetoencephalography.md
-        - Electroencephalography: modality-specific-files/electroencephalography.md
-        - Intracranial Electroencephalography: modality-specific-files/intracranial-electroencephalography.md
-        - Task events: modality-specific-files/task-events.md
-        - Physiological and other continuous recordings: modality-specific-files/physiological-and-other-continuous-recordings.md
+      - Modality specific files: # keep sections in sorted order when adding/editing
         - Behavioral experiments (with no neural recordings): modality-specific-files/behavioral-experiments.md
+        - Electroencephalography: modality-specific-files/electroencephalography.md
         - Genetic Descriptor: modality-specific-files/genetic-descriptor.md
-        - Positron Emission Tomography: modality-specific-files/positron-emission-tomography.md
+        - Intracranial Electroencephalography: modality-specific-files/intracranial-electroencephalography.md
+        - Magnetic Resonance Imaging: modality-specific-files/magnetic-resonance-imaging-data.md
+        - Magnetoencephalography: modality-specific-files/magnetoencephalography.md
         - Microscopy: modality-specific-files/microscopy.md
         - Near-Infrared Spectroscopy: modality-specific-files/near-infrared-spectroscopy.md
-      - Derivatives:
+        - Physiological and other continuous recordings: modality-specific-files/physiological-and-other-continuous-recordings.md
+        - Positron Emission Tomography: modality-specific-files/positron-emission-tomography.md
+        - Task events: modality-specific-files/task-events.md
+      - Derivatives: # keep sections in sorted order when adding/editing
```

Comment:
I guess it's lucky that derivatives were already sorted. But these are also sorted from "most broadly applicable" to "most specific".

Reply:
yeah -- here may be we could leave as is without comment